.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: java
+before_cache: 
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+
+before_install: 
+  - echo "artifactory_username=$ARTIFACTORY_USERNAME" > gradle.properties
+  - echo "artifactory_password=$ARTIFACTORY_PW" >> gradle.properties
+  - echo "artifactory_contextURL=https://blockone.jfrog.io/blockone" >> gradle.properties
+  - echo "artifactory_repo=android-libs-scratch-local" >> gradle.properties
```

Comment:
we should use android-libs-dev-local for this. That value is specified by the branch. 
Example: 
local: android-libs-scratch-local
feature: android-libs-feature-local
develop: android-libs-dev-local
master: android-libs-release-local

Reply:
From Nelson and Shawn: the handling of the field will be handled on a different story.

Reply:
Thanks for the review!